### DISCUSSION TITLE
How to deploy a project which has npm packages on GitHub pages?

### DISCUSSION THREAD

**Discussion by f-mohrechi** [community/community#54049] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have a project built with next.js and it has a few npm packages.
I want to deploy it on GitHub pages, but GitHub pages ignore my packages, and deploy the pure HTML, CSS, and javascript.
Do you guys know how I can make it?

**Comment by YasinDehfuli** [comment, marked as answer]
As you can see in https://github.com/orgs/community/discussions/categories/pages title bar :

Conversations related to GitHub Pages, a static site hosting service that takes HTML, CSS, and JavaScript files straight from a repository on GitHub and publishes a website.

So You Can not add packages in Github Pages by add package or something like this . 

but i use some ways to add packages and use it in Pages . 
you can easily Link CDN link of your package you need in HTML . then you can see your View Deployment Page will work carefully .

As a example . if you installed Bootstrap V5.1 npm package in your files . you can easily use this link to find Bootstrap V5.1 cnd link . and link it in your HTML <head> link : 
https://cdnjs.com/libraries/bootstrap/5.0.1

I hope it is helpful for you !
